docs/en_US/NAS/SearchSpaceZoo.md

```diff
@@ -0,0 +1,160 @@
+# Search Space Zoo
+
+## DartsCell
+
+DartsCell is extracted from [CNN model](./DARTS.md) designed in this repo. [Operations](#darts-predefined-operations) connecting with nodes which contained in the cell strucure is fixed.
+
+The predefined operations are shown as follows:
+
+* MaxPool: call `torch.nn.MaxPool2d`. This operation applies a 2D max pooling over all input channels. Its parameters `kernal_size=3` and `padding=1` are fixed.
+* AvgPool: call `torch.nn.AvgPool2d`. This operation applies a 2D average pooling over all input channels. Its parameters `kernal_size=3` and `padding=1` are fixed.
+* Skip Connect: There is no operation between two nodes. Call `torch.nn.Identity` to forward what it gets to the output.
+* SepConv3x3: Composed of two DilConvs with fixed `kernal_size=3` sequentially.
+* SepConv5x5: Do the same operation as the previous one but it has different kernal size, which is set to 5.
+* <a name="DilConv"></a>DilConv3x3:  (Dilated) depthwise separable Conv. It first calls `torch.nn.Conv2d` with fixed `kernal_size=3` to partition the feature map into `C_in` groups then applies 1x1 Convolution to get `C_out` output channels. It makes extracting features on every channel separately possible and reduces the number of parameters.
```

Comment:
"It makes extracting features on every channel separately possible". Are you sure this is a correct sentence?